obs-browser-source.cpp

```diff
@@ -633,7 +633,14 @@ void BrowserSource::Render()
 
 	if (texture) {
 #ifdef __APPLE__
-		gs_effect_t *effect = obs_get_base_effect((hwaccel) ? OBS_EFFECT_DEFAULT_RECT : OBS_EFFECT_DEFAULT);
+		int type = gs_get_device_type();
```

Comment:
It might be more efficient to store the device type on initialization than calling this here in `Render`, but perhaps that can be changed later if there's any measurable impact.